docs/src/pages/careers/lead-designer.md

```diff
@@ -69,10 +70,9 @@ As a **team leader**, you will:
 
 We're looking for someone with an expert eye for both UI and UX design. Details matter to you.
 
-You're a leader who can support and guide a small but growing design team.
+You're a manager that can support and guide the team.
 
-It's also important that you have enough development skills – CSS, JSX, Git, etc. – to be accountable for the quality of the design experience implemented.
-You will collaborate with engineers, product managers, and other designers, to ensure that the implementation truly reflects the feature/project's vision.
+And, it's also important that you have enough development skills – CSS, JSX, Git, etc. – to communicate effectively with engineers, product managers, and other designers, ensuring that the implementation truly reflects the feature/project's vision.
```

Comment:
I propose this alternative version. The big change would be: 1. the person in this role will be accountable for the end experience of the design, the implementation **included**. 2. we shift from "communication" to "collaboration".

```suggestion
It's also important that you have enough development skills – CSS, JSX, Git, etc. – to be accountable for the quality of the design experience implemented.
You will collaborate with engineers, product managers, and other designers, to ensure that the implementation truly reflects the feature/project's vision.
```